docs/provider/provider-features.mdx

```diff
@@ -231,10 +231,24 @@ lavap tx pairing stake-provider "ETH1" \
     --chain-id {CHAIN_ID} \
     --gas="auto" \
     --gas-adjustment "1.5" \
-    --node {PUBLIC_RPC}
+    --node {PUBLIC_RPC} \
+    --delegation-limit 0ulava
 ```
 
+The `delegation-limit` flag is mandatory (but will be removed in future versions). The delegation limit is the maximum amount of delegations the provider is willing to use for the pairing process.
+Larger delegations mean the provider will be paired more often with consumers. If a provider receives many delegations but can't handle the resulting consumer traffic, they can set the delegate limit lower than their actual total delegations. This reduces their pairing frequency and workload. The delegated tokens remain with the provider, but their influence is reduced by artificially lowering the provider's effective stake (composed of self-stake and delegations).
+Putting "0ulava" means there are no delegations restrictions.
```

Comment:
this is not true, 0 means there is a strict restriction to no delegations affecting pairing